lib/main_ltp.pm

```diff
@@ -93,12 +95,22 @@ sub loadtest_from_runtest_file {
         loadtest 'create_junkfile_ltp';
     }
 
+    if (!-d $unpack_path) {
+        mkdir($unpack_path, 0755);
+    }
+    chdir($unpack_path);
```

Comment:
Does this reset the working path if we die before changing it back?

Reply:
No. OK, if this is needed because tests can be restarted I'll fix it.

Reply:
I fixed this by setting path for tar, so we don't change cwd for the test.